modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIEndpointRelevantObjectEntryModelListenerTest.java

```diff
@@ -156,39 +131,6 @@ public void testCreateGetAPIEndpoint() throws Exception {
 			).toString(),
 			"headless-builder/applications", Http.Method.POST);
 
-		JSONAssert.assertEquals(
```

Comment:
Why did you delete this test scenario? You can substitute `RandomTestUtil.nextLong()` with `_objectEntry.getId()`

Reply:
I did it in another test, that's why this one seemed redundant 😅

Reply:
But be careful, because the error messages are not the same :smile:

Reply:
added it back, thanks for catching it! :)